docs/synchronizations-faq.md

```diff
@@ -0,0 +1,27 @@
+# Synchronizations FAQ
+
+In Private Packagist synchronizations is how you keep packages, teams, their members, and access permissions in sync with your version control system like GitHub, GitLab or Bitbucket.
+
+You can find below answers for frequently-asked questions concerning Synchronizations.
+
+## How to migrate my synchronization from GitHub to Bitbucket?
+
+1. Add the new synchronization from Bitbucket.
+2. Delete the packages that were created from the GitHub synchronization. You can filter by the GitHub synchronization on the package list page and then you can delete the packages from that list.
+3. Make sure the new synchronization from Bitbucket successfully run. Then you can safely delete the old synchronization from GitHub.
```

Comment:
This should probably also mention that the primary sync needs to be updated and what this implies? Same for the questions below

Reply:
what do you mean by the primary sync needs to be updated? is this in case the old sync was the primary one?

Reply:
Right, if the old sync was the primary sync then before you can delete it you need to promote another sync as primary